docs/reference/cluster/update-settings.asciidoc

```diff
@@ -88,25 +98,19 @@ PUT /_cluster/settings
 --------------------------------------------------
 // CONSOLE
 
-Cluster wide settings can be returned using:
-
-[source,js]
---------------------------------------------------
-GET /_cluster/settings
---------------------------------------------------
-// CONSOLE
 
 [float]
-=== Precedence of settings
+=== Order of Precedence
+
+The order of precedence for cluster settings is:
 
-Transient cluster settings take precedence over persistent cluster settings,
-which take precedence over settings configured in the `elasticsearch.yml`
-config file.
+1. transient cluster settings 
+2. persistent cluster settings
+3. settings in the `elasticsearch.yml` configuration file.
 
-For this reason it is preferrable to  use the `elasticsearch.yml` file only
-for local configurations, and set all cluster-wider settings with the
+It's best to use the `elasticsearch.yml` file only
```

Comment:
Perhaps expand on this: the reason why it is better to use cluster settings instead of settings in the config file is that you can be sure that the setting is the same on all nodes.  If different nodes have different settings, and you don't use cluster settings, there is no way of knowing about these discrepancies.